ghost/i18n/lib/i18n.js

```
@@ -84,18 +86,90 @@ function generateResources(locales, ns) {
+
+    return locales.reduce((acc, locale) => {
+        let res;
+        // Try to load the locale file, fallback to English
+        try {
+            const localePath = path.join(themeLocalesPath, `${locale}.json`);
+            if (fs.existsSync(localePath)) {
+                // Delete from require cache to ensure fresh reads for theme files
+                delete require.cache[require.resolve(localePath)];
+                res = require(localePath);
+            } else {
+                throw new Error(`Locale file not found: ${locale}`);
```

ESLint

[error] Direct calls to new Error() are not allowed. Please use @tryghost/errors.

(ghost/ghost-custom/no-native-error)